.github/workflows/ci.yaml

```diff
@@ -76,6 +76,13 @@ jobs:
           echo "run KWIL-JS TEST against kwild(http:localhost:8484),  with $chain_id"
           PRIVATE_KEY=0000000000000000000000000000000000000000000000000000000000000001 CHAIN_ID=$chain_id GATEWAY_ON=false GAS_ON=false KWIL_PROVIDER=http://localhost:8484 npm run integration
       
+      - name: Run integration tests in private mode
```

Comment:
Double check this. I don't think you are restarting kwild in private mode. It looks like you're just running the kwil-js tests again with the private mode flag set to true.